raft/src/test/java/org/apache/kafka/raft/QuorumStateTest.java

```diff
@@ -204,37 +208,53 @@ public void testCannotBecomeLeaderWithoutGrantedVote() throws IOException {
     public void testLeaderToFollowerOfElectedLeader() throws IOException {
         int otherNodeId = 1;
         Set<Integer> voters = Utils.mkSet(localId, otherNodeId);
-        QuorumState state = new QuorumState(localId, voters, store, new LogContext());
-        state.initialize(new OffsetAndEpoch(0L, logEndEpoch));
+
+        QuorumState state = initializeState(voters, store);
+
         state.becomeCandidate();
         state.candidateStateOrThrow().recordGrantedVote(otherNodeId);
         state.becomeLeader(0L);
         assertTrue(state.becomeFetchingFollower(5, otherNodeId));
         assertEquals(5, state.epoch());
         assertEquals(OptionalInt.of(otherNodeId), state.leaderId());
-        assertEquals(ElectionState.withElectedLeader(5, otherNodeId), store.readElectionState());
+        assertEquals(ElectionState.withElectedLeader(5, otherNodeId, voters), store.readElectionState());
+    }
+
+    private QuorumState initializeState(Set<Integer> voters,
```

Comment:
To be clearer, could could we call this `initializeEmptyState` or something like that?

Reply:
Make sense!